### ISSUE TITLE
Cast lure if bobber not found x number of times

### ISSUE THREAD

**Issue by phrost401** [jsbots/AutoFish#65]
Could you add an option for when we have lures enabled to have it recast lure if the bobber is not found for x number of casts, instead of stopping playback. Would be great sometimes the timing is a little off and it doesn't recast, causing the bobber not to cast in high level fishing zones that are unfishable without lures. Thanks :)

**Comment by phrost401** [comment]
update: i figured out i could update the macro for the lure and make it look like this:

```
/equip Big Iron Fishing Pole
/use Aquadynamic Fish Attractor
/use 16
/click StaticPopup1Button1
```

The last part clicks Yes to apply the lure if it is already on, i have not had an issue since.

**Comment by jsbots** [comment]
solved in 2.6.0
